### DISCUSSION TITLE
Sprint 0 - Team retrospective

### DISCUSSION THREAD

**Discussion by lukaskarlsson290** [cs481-ekh/f22-dust-devils#19] [Retrospectives]
This was definitely a slower sprint in terms of our progress. As a group, it felt as though we weren't clear on the goals of the project. There was a lot of information thrown at us, so we took the approach of meeting with the sponsor initially to get an idea of what he's working on and doing.
After this, we came together and did the necessary in-class work to setup for the project. We had to refresh ourselves on how to work in a team on GitHub and the new functionality that's in place. 
We decided we needed some more clarity, so we met with Dr. Jackson to get a clear outline and put together the necessary Epics for this project. We feel much better and that we have a clear picture of what we need to do going forward. 

**Roles in Sprint 0:**
Product Owner - Samantha
Project Lead - Sena
Dev - Jim
Scribe - Lukas

What wasn't completed:
- [ ] Cloned repo - Sena: pull request is ready
- [ ] Setup linter - Lukas: did not get to it
- [ ] Test product on team side - Jim: did not get to it

Our goal is to complete the above items as soon as possible. Now that we're more organized and have more clarity on the project, we're in a much better position for Sprint 1.

**Comment by ekhenderson** [comment]
Good writeup!
